Source/Core/DolphinQt2/NetPlay/MD5Dialog.h

```diff
@@ -32,7 +32,7 @@ class MD5Dialog : public QDialog
   std::map<int, QProgressBar*> m_progress_bars;
   std::map<int, QLabel*> m_status_labels;
 
-  std::string m_last_result;
+  std::vector<std::string> m_results;
```

Comment:
`<vector>` needs to be included (as well as `<map>` and `<string>` which should have been included even before this change).

Reply:
Why does it compile without those?

Reply:
Because of other inclusions satisfying those dependencies in the files this header file is included in (which shouldn't be relied upon).

e.g.

```cpp
// SomeFile.cpp

#include <map>
#include <string>

// Won't cause a compilation error, dependencies are satisfied indirectly
#include "DolphinQt2/NetPlay/MD5Dialog.h"

// ... rest of the code
```